lib/flv/transmuxer.js

```diff
@@ -400,6 +400,13 @@ Transmuxer = function(options) {
     packetStream.flush();
   };
 
+  // Caption data has to be reset when seeking outside buffered range
+  this.resetCaptions = function() {
+    if (this.transmuxPipeline_.captionStream) {
```

Comment:
looks like the flv transmuxer doesnt have a `this.transmuxPipeline_` this can just be updated to
```
this.resetCaptions = function() {
  captionStream.reset();
};
```